### ISSUE TITLE
Can F-Stack run on Ryzen CPU?

### ISSUE THREAD

**Issue by Onepamopa** [F-Stack/f-stack#445]
Hello,

Can F-Stack run on Ryzen CPUs? Let me know.

**Comment by jfb8856606** [comment]
X86-64 only now.

**Comment by Onepamopa** [comment]
Isn't Ryzen a x86-64 CPU ??

**Comment by Onepamopa** [comment]
Has anyone ran F-Stack using an amd ryzen or threadripper CPU?

**Comment by jfb8856606** [comment]
Yes. AMD Ryzen and Threadripper are x86-64 CPUs, and F-Stack only requires the x86-64 architecture — it does not care about the Intel/AMD vendor. The earlier "x86-64 only" reply just meant "x86-64 architecture (not ARM, etc.)", it was not meant to exclude AMD.

What actually matters for running F-Stack is **not the CPU brand** but:
1. Your NIC must be supported by a DPDK PMD (Intel, Mellanox, and several virtual NICs are supported — check the DPDK supported-NICs list);
2. The platform must support **IOMMU (AMD-Vi) + hugepages** for VFIO passthrough.

So on a Ryzen/Threadripper box with a DPDK-supported NIC, F-Stack runs fine. Closing this — feel free to reopen if you hit a specific issue on your AMD setup.
